drivers/i2c/Makefile

```diff
@@ -7,5 +7,7 @@ obj-$(CONFIG_I2C_NRF5) += i2c_nrf5.o
 obj-$(CONFIG_I2C_QMSI) += i2c_qmsi.o
 obj-$(CONFIG_I2C_QMSI_SS) += i2c_qmsi_ss.o
 obj-$(CONFIG_I2C_SBCON) += i2c_sbcon.o
-obj-$(CONFIG_I2C_STM32LX) += i2c_stm32lx.o
+obj-$(CONFIG_I2C_STM32_V2) += i2c_ll_stm32_v2.o i2c_ll_stm32.o
```

Comment:
Have this bee:

obj-$(CONFIG_I2C_STM32) += i2c_ll_stm32.o
obj-$(CONFIG_I2C_STM32_v1) += i2c_ll_stm32_v1.o
obj-$(CONFIG_I2C_STM32_v2) += i2c_ll_stm32_v2.o